src/kernels/jupyter/serverSelector.ts

```diff
@@ -663,3 +676,288 @@ class JupyterServerSelector_Original implements IJupyterServerSelector {
         return items;
     }
 }
+
+/**
+ * Inisders version of the JupyterServerSelector.
+ */
+class JupyterServerSelector_Insiders implements IJupyterServerSelector {
```

Comment:
This is the new insiders version of the server selector, it's basically the same as the Original version (not 'Experimental'). Don't love the juggling here, but to prototype + work with stable and the previous experiment this seems the best way. I'll call out the different areas in this PR, but it's 98% the same.